webplugin/js/app/sentry-8.39.0.js

```
@@ -0,0 +1,331 @@
+        }
+        try {
+            // We have to make sure to call all callbacks first
+            for (var i = 0; i < onLoadCallbacks.length; i++) {
+                if (typeof onLoadCallbacks[i] === 'function') {
+                    onLoadCallbacks[i]();
+                }
+            }
+            // Cleanup to allow garbage collection
+            onLoadCallbacks.splice(0);
+            // First call all inits from the queue
+            for (var i = 0; i < queue.length; i++) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)